### PR TITLE
996: Add T2A cases and calculate the totals

### PR DESCRIPTION
**Pull request by ddebella**

### CODE REVIEW

File: test/unit/context-map/test-workload.js [hunk 1]
```diff
@@ -46,10 +53,12 @@ describe('context-map/workload', function () {
...
       workloadWith24Cases.casesSummary.custodyTiers = stagingHelper.getCountableTestTiers(Locations.CUSTODY)
       workloadWith24Cases.casesSummary.licenseTiers = stagingHelper.getCountableTestTiers(Locations.LICENSE)
+      workloadWith24Cases.casesSummary.t2aCommunityTiers = stagingHelper.getCountableTestTiers(Locations.COMMUNITY)
+      workloadWith24Cases.casesSummary.t2aCustodyTiers = stagingHelper.getCountableTestTiers(Locations.CUSTODY)
+      workloadWith24Cases.casesSummary.t2aLicenseTiers = stagingHelper.getCountableTestTiers(Locations.LICENSE)
 
       var mappedWorkloadWith24Cases = mapper(workloadWith24Cases, ownerId, workloadReportId)
-
-      expect(mappedWorkloadWith24Cases.totalCases).to.equal(24)
+      expect(mappedWorkloadWith24Cases.totalCases).to.equal(48)
```
Comment (andrew-js-wright) [inline test/unit/context-map/test-workload.js]: Minor: This workload now has 48 cases so `mappedWorkloadWith24Cases` should be updated.

File: test/unit/context-map/test-workload.js [hunk 2]
```diff
@@ -88,7 +97,21 @@ describe('context-map/workload', function () {
       expect(mappedWorkload.licenseTiers.c1.total).to.eq(parseInt(licenseTiers.c1))
       expect(mappedWorkload.licenseTiers.b2.total).to.eq(parseInt(licenseTiers.b2))
       expect(mappedWorkload.licenseTiers.b1.total).to.eq(parseInt(licenseTiers.b1))
-      expect(mappedWorkload.licenseTiers.a.total).to.eq(0)      
+      expect(mappedWorkload.licenseTiers.a.total).to.eq(0)
+    })
+
+    it('should correctly calculate the total number of t2a cases', function () {
```
Comment (andrew-js-wright) [inline test/unit/context-map/test-workload.js]: 👍

File: test/unit/context-map/test-workload.js (hunk 1 above)
Comment (ddebella) [reply test/unit/context-map/test-workload.js]: good catch!

**Comment by caoimheb** [comment]
Version will need bumped and master merged and conflicts resolved, but apart from that, LGTM

**Pull request closed without merging**